kafl_fuzzer/common/config.py

```diff
@@ -247,22 +248,31 @@ def _parse_with_config(self, parser):
 
         config = confuse.Configuration('kafl', modname='kafl_fuzzer')
 
-        # check default config search paths
+        # check default config locations
         config.read(defaults=True, user=True)
 
+        # ENV based config
+        if 'KAFL_CONFIG_FILE' in os.environ:
+            try:
+                debug_print(f"Checking configuration at KAFL_CONFIG_FILE: {os.environ['KAFL_CONFIG_FILE']}")
+                config.set_file(os.environ['KAFL_CONFIG_FILE'])
+            except Exception as e:
+                logger.error("Failed to read configuration at $KAFL_CONFIG_FILE:\n" + str(e))
```

Comment:
inside an exception handler, an seeing that you want to print the actual error string, you should use [`logger.exception()`](https://docs.python.org/3/library/logging.html#logging.Logger.exception)

Reply:
This is not logger...see https://github.com/IntelLabs/kafl.fuzzer/issues/1